site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -989,17 +989,19 @@ function () use ($csv_file) {
         $row_num = 1;
         $error_message = "";
         $new_teams_members = [];
+        $team_names = [];
         foreach ($contents as $content) {
             $vals = str_getcsv($content);
             $vals = array_map('trim', $vals);
-            if (count($vals) != 6) {
+            if (count($vals) != 7) {
```

Comment:
Given that `$team_name` can be null, I would make this accept both the new format that includes team name and old format which does not to retain BC compatibility.

Reply:
I think if we go this route we will need to rely on column names rather than hard coded column numbers so we can check what exists and what does not. We could keep it hard coded column numbers but I think if we add another feature to teams with another column it may get messy. Thoughts?

Reply:
Fair enough, screw BC then.